Dockerfile

```diff
@@ -36,8 +25,8 @@ COPY ./api/requirements.txt api/requirements.txt
 RUN pip install --upgrade pip && \
     pip install --no-cache-dir -r ./api/requirements.txt
 
-# Copy files
-COPY --from=llama_builder /tmp/llama.cpp/llama /usr/bin/llama
+COPY compile.sh .
```

Comment:
Move this COPY after the WORKDIR, then do the "chmod" together with the apt-install commands in the same RUN

Reply:
I just got rid of the RUN chmod :stuck_out_tongue: